doc/changes/changes49.xml

```diff
@@ -1206,4 +1206,90 @@ so that it can be distributed again, please contact the &GAP; team.
 
 </Section>
 
+<Section Label="gap492">
+<Heading>&GAP; 4.9.2 (July 2018)</Heading>
+
+<Subsection Label="Changes in the core GAP system introduced in GAP 4.9.2">
+<Heading>Changes in the core &GAP; system introduced in &GAP; 4.9.2</Heading>
+
+Fixed bugs that could lead to break loops:
+<List>
+<Item>
+Fixed a bug in iterating over an empty cartesian product
+(<URL><LinkText>&Hash;2421</LinkText><Link>https://github.com/gap-system/gap/pull/2421</Link></URL>).
+[Reported by @isadofschi]
+</Item>
+</List>
+
+Fixed bugs that could lead to crashes:
+<List>
+<Item>
+Fixed a crash after entering <C>return;</C> in a <Q>method not found</Q> break loop
+(<URL><LinkText>&Hash;2449</LinkText><Link>https://github.com/gap-system/gap/pull/2449</Link></URL>).
+</Item>
+<Item>
+Fixed a crash when an error occurs and
+<Ref BookName="ref" Oper="OutputLogTo" Label="for streams"/>
+points to a stream which uses a stream
```

Comment:
Optional: "a stream which uses a stream"? -> I know this comes from the description of the issue, but I find it highly confusing. Perhaps "points to a stream which internally uses another stream"?